src/libs/ReportUtils.js

```diff
@@ -1072,7 +1072,7 @@ function getChatRoomSubtitle(report) {
         }
 
         const workspaceName = getPolicyName(report);
-        return `${Localize.translateLocal('threads.from')} ${roomName ? [roomName, workspaceName].join(' in ') : workspaceName}`;
+        return `${Localize.translateLocal('threads.from')} ${roomName ? [roomName, workspaceName].join(` ${Localize.translateLocal('threads.in')} `) : workspaceName}`;
```

Comment:
Let's take the opportunity _now_ to fix this fully and combine the phrase into 1 translation instead of piecing the words together. Thanks!

Reply:
Addressed 👍 thanks for bringing it to my attention

Reply:
oh i thought this was my PR, lol. let's close out this PR since I'm tackling it [here](https://github.com/Expensify/App/pull/21092)